extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -238,6 +239,7 @@ export function waitForNewHostFile(): IFileFunctionObject {
  * @param selectedProjectFile is the users selected project file path
  */
 export async function addSqlNugetReferenceToProjectFile(selectedProjectFile: string): Promise<void> {
+	TelemetryReporter.createActionEvent(TelemetryViews.CreateAzureFunctionWithSqlBinding, AzureFunctionUtils.addSQLNugetPackage).send();
```

Comment:
You can just call `sendActionEvent` directly, no need to create then send when there aren't custom properties/measurements.